karabo_data/tests/mockdata/basler_camera.py

```diff
@@ -0,0 +1,38 @@
+"""Script that creates a mock-run for the basler camera"""
+from os.path import join as op_join
+
+from .base import DeviceBase
+
+class BaslerCamera(DeviceBase):
+    """
+    Basler Camera device
+    Based on example /gpfs/exfel/exp/SPB/201930/p900061/raw/r0055/RAW-R0055-DA01-S00000.h5
+    """
+
+    def __init__(self, device_id, nsamples=None, px_size=None):
+        """Create a dummy basler device that inherits from Device Base"""
+        self.px_size = px_size or (2058, 2456)
+        super(BaslerCamera, self).__init__(device_id, nsamples=nsamples)
+        self.output_channels = ('daqOutput/data',)
```

Comment:
Can you add back the comment from @takluyver 
```python
# Technically, only the part before the / is the output channel.
# But there is a structure associated with the part one level after that,
# and we don't know what else to call it.
```

Reply:
Okidoke